packages/caliper-vscode/extension/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "caliper-vscode-extension",
+	"displayName": "caliper-vscode-extension",
+	"description": "",
```

Comment:
This should be filled with a single, short, descriptive sentence about the extension. Look up some extensions in vscode to get a sense of what kinds of descriptions should be used.